torch/_inductor/codegen/common.py

```diff
@@ -635,6 +648,12 @@ def generate(
                 else:
                     line = f"{expr}{self.suffix}"
                 buffer.writeline(line)
+                if assignment and name == "constant" and origin_args[1] in _uint_types:
+                    target_tl_type = _torch_uint_type_to_tl_type(origin_args[1])
+                    line = (
+                        f"{self.prefix}{var} = {self.prefix}{var}.to({target_tl_type})"
```

Comment:
Things under common.py are also used by cpp.py, so you can't do any triton specific type conversion here.

Reply:
fixed in [remove changes in common.py; fix test unit](https://github.com/pytorch/pytorch/pull/101468/commits/8f6f6736e09b7bd08c23a99fbfbe6e68abdb9f23)